File: Dataset/cpp/commanding.cpp

//  commanding.cpp (c) 1998-2002 Kari Laitinen

#include  <iostream.h>

int main( int    number_of_command_line_arguments,
          char*  command_line_arguments[] )
{
   cout << "\n Function main was given the following "
        << number_of_command_line_arguments << " strings"
        << "\n as command line arguments: \n" ;

   int  argument_index  =  0 ;

   while ( argument_index  <  number_of_command_line_arguments )
   {
      cout << "\n     "
           << command_line_arguments[ argument_index ] ;

      argument_index  ++ ;
   }
}

File: Dataset/cs/Commanding.cs

//  Commanding.cs (c) 2003 Kari Laitinen

using System  ;

class  Commanding
{
   static void Main( string[]  command_line_parameters )
   {
      Console.Write( "\n Method Main() was given the following "
                  +  command_line_parameters.Length + " strings"
                  +  "\n as command line parameters: \n" ) ;

      int  parameter_index  =  0 ;

      while ( parameter_index  <  command_line_parameters.Length )
      {
         Console.Write(
             "\n     " + command_line_parameters[ parameter_index ] ) ;

         parameter_index  ++ ;
      }
   }
}




